include/dev/rtl8139_reg.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _RTL8139_REG_H_
+#define _RTL8139_REG_H_
+/*
+ * Copied from FreeBSD: sys/dev/rl/if_rlreg.h
+ */
+
+#define RL_COMMAND 0x0037 /* command register */
+
+#define RL_8139_CFG1 0x0052 /* config register #1 */
+
+/*
+ * Command register.
+ */
+#define RL_CMD_RESET 0x0010
+
+/*
+ * Config 1 register
+ */
+#define RL_CFG1_PWRON 0x00
```

Comment:
Why not just `RL_CFG1` and `RL_CFG1_...`? Do you really want to literally copy all definitions from their header file? Maybe they have other devices starting with prefix "RL" and hence they use `RF_8139_CFG1` instead of `RF_CFG1`.

Reply:
It was idea to copy whole file and do not introduce changes wherever possible